tests/mobile-conf.js

```diff
@@ -0,0 +1,22 @@
+const constants = require('./constants');
```

Comment:
This is also defined in grunt file and seems to be redefined here. I think we can get rid of the extra conf file and use the existing one. 

Define the new mobile suite in the `tests/conf.js` and pass the arguments in the grunt file. Which I believe you're already doing.

```
      'e2e-tests-mobile': {
        options: {
          configFile: 'tests/mobile-conf.js',
          args: {
            suite: 'mobile'
          },
          capabilities: {
            chromeOptions: {
              mobileEmulation: { 'deviceName': 'Nexus 5' }
            }
          }
        }
      }
```